src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -130,6 +130,7 @@ const CoverImageEditModal = ({
           "Bearer " + localStorage.getItem(LocalStorageKeys.accessToken),
       },
       async (xhr: XMLHttpRequest) => {
+        const errorResponse = JSON.parse(xhr.responseText);
```

Comment:
can we parse this only in the else block?

(i.e. the status is non 2xx)


```suggestion
        const error = JSON.parse(xhr.responseText);
```

Reply:
hello @rithviknishad  should i make it like this 
 else {
    try {
      const error = JSON.parse(xhr.responseText); 
      Notification.Error({
        msg: errorResponse?.cover_image?.join(' '),
      });
    } 
    /// this is for if parsing err fails 
    catch (e) {
      Notification.Error({
        msg: "Something went wrong!",
      });
    }